client/src/RatingsReviews/Ratings/CharBar.jsx

```diff
@@ -6,15 +6,16 @@ import ratingToPercentage from '../lib/ratingToPercentage';
 function CharBar({ rating, char, chars }) {
   return (
     <BarContainer>
-      <StarCount>{char}: </StarCount>
+      <StarCount>
+        {char}
+        :
+        {' '}
+      </StarCount>
       <OuterBar>
-        <InnerBar width={ratingToPercentage(rating)}>
-        </InnerBar>
+        <InnerBar width={ratingToPercentage(rating)} />
       </OuterBar>
       <CharList>
-        {chars.map((c, index) => {
-          return <Char char={c} key={index}/>
-        })}
+        {chars.map((c, index) => <Char char={c} key={index} />)}
```

Comment:
if char is unique within the map, you could use char as a key, since using an index value for key is not recommended (by eslint?)